md/02.QuickStart/Ollama_QuickStart.md

```diff
@@ -113,17 +113,45 @@ ollama run phi3
 ***Note:*** Visit this link [https://github.com/ollama/ollama/blob/main/docs/api.md](https://github.com/ollama/ollama/blob/main/docs/api.md) to learn more
 
 
+## Calling Ollama from Python
+
+You can use `requests` or `urllib3` to make requests to the local server endpoints used above. However, a popular way to use Ollama in Python is via the [openai](https://pypi.org/project/openai/) SDK, since Ollama provides OpenAI-compatible server endpoints as well.
+
+Here is an example for phi3-mini:
+
+```python
+import openai
+
+client = openai.OpenAI(
+    base_url="http://localhost:11434/v1",
+    api_key="nokeyneeded",
+)
+
+response = client.chat.completions.create(
+    model="phi3"
+    temperature=0.7,
+    n=1,
+    messages=[
+        {"role": "system", "content": "You are a helpful assistant."},
+        {"role": "user", "content": "Write a haiku about a hungry cat"},
+    ],
+)
+
+print("Response:")
+print(response.choices[0].message.content)
+```
+
 ## Calling Ollama from JavaScript 
 
 ```javascript
-#Example of Summarize a file with Phi-3
+// Example of Summarize a file with Phi-3
```

Comment:
I presume these should be // since its JS, but I also don't understand how this code is supposed to work.. Where does script come from?? Is it a network call? This seems incomplete?